source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/AdaptiveCardRenderer.java

```diff
@@ -255,6 +260,15 @@ else if (alignment == ActionAlignment.Center.swigValue())
             ActionElementRenderer.getInstance().render(renderedCard, context, fragmentManager, actionButtonsLayout, actionElement, cardActionHandler, hostConfig);
         }
 
+        if (viewGroup != null && hostConfig.getActions().getActionsOrientation().swigValue() == ActionsOrientation.Horizontal.swigValue())
+        {
```

Comment:
We are adding actionbuttonslayout above and then removing it from view group. Can you change the code above to selectively add to scroll view if required and avoid the add and removal

Reply:
@rednivrug15  I agree with this comment, can you change it?

Reply:
@almedina-ms  Since the horizontal scroll has been implemented I will revert the change for horizontal scroll and keep rest of the bug fixes here . Can you please look into this ?